### DISCUSSION TITLE
通过浏览器访问BE节点（8040、8060端口）可否配置访问认证？

### DISCUSSION THREAD

**Discussion by ShadowHotz** [apache/doris#20254] [A - General / Q&A]
最近在政务云部署doris后，收到云服务商的漏洞提醒，如下图，请问通过浏览器访问BE节点有访问认证相关的配置吗？
![image](https://github.com/apache/doris/assets/15317352/9e126b4e-1427-4e25-8d68-f2661652983f)
![image](https://github.com/apache/doris/assets/15317352/6dbcdf22-e82e-43d6-b99d-61af7ae67250)

**Comment by LemonLiTree** [comment]
not yet

**Reply by ysworld1111** [reply]
请问问题有找到解决方案吗

**Reply by ShadowHotz** [reply]
最后是通过配置ip白名单限制访问解决的，当时联系他们说在新版版里会加上

**Comment by ysworld1111** [comment]
好的，谢谢指导


| |
亦诗
|
|
***@***.***
|


---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年10月21日 09:42 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [apache/doris] 通过浏览器访问BE节点（8040、8060端口）可否配置访问认证？ (Discussion #20254) |

最后是通过配置ip白名单限制访问解决的，当时联系他们说在新版版里会加上
